webapp/components/Comment/Comment.vue

```diff
@@ -141,6 +151,10 @@ export default {
     },
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-09T07:48:47Z" title="Thursday, January 9th 2020, 8:48:47 am +01:00">Jan 9, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -168,6 +176,23 @@ export default {
         this.$toast.error(err.message)
       }
     },
+    answerComment() {
+      const slug =
+        '<a class="mention" href="/profile/' +
+        this.comment.author.id +
+        '" data-mention-id="' +
+        this.comment.author.slug +
+        '" target="_blank" contenteditable="false">@d' +
```


The `@d` should be just `@` I would say.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-10T07:20:37Z" title="Friday, January 10th 2020, 8:20:37 am +01:00">Jan 10, 2020</time>_

---


ok

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-10T19:15:43Z" title="Friday, January 10th 2020, 8:15:43 pm +01:00">Jan 10, 2020</time>_

---


Wow, now I get what's going on. Please use VueJS events!

![so dirty](https://media.giphy.com/media/oHwwRJnhYoU22WMBO2/giphy.gif)